t/api/13-influxdb.t

```diff
@@ -67,6 +67,28 @@ $t->get_ok('/admin/influxdb/minion')->status_is(200)->content_is(
 openqa_minion_workers,url=http://example.com active=0i,inactive=1i
 "
 );
+
+subtest 'filter specified minion tasks' => sub {
+    for my $task (qw(obs_rsync_run obs_rsync_update_builds_text)) {
+        $t->app->minion->add_task($task => sub { });
+        my $job_ids = $t->app->minion->enqueue($task);
+    }
+    while (my $tmp_job = $worker->dequeue(0)) { $tmp_job->fail('this is a test'); }
+
+    $t->get_ok('/admin/influxdb/minion')->status_is(200)->content_is(
+        "openqa_minion_jobs,url=http://example.com active=0i,delayed=1i,failed=3i,inactive=1i
+openqa_minion_workers,url=http://example.com active=0i,inactive=1i
+"
+    );
+
+    $t->app->config->{global}->{influxdb_minion_job_blocklist} = 'obs_rsync_run obs_rsync_update_builds_text';
+    $t->get_ok('/admin/influxdb/minion')->status_is(200)->content_is(
+        "openqa_minion_jobs,url=http://example.com active=0i,delayed=1i,failed=1i,inactive=1i
```

Comment:
So this contains `failed=1i` in place of `failed=3i`. It's very difficult to even see that they are different. Could you please add a comment pointing out the difference and why this is correct?

Reply:
added